netlify.toml

```diff
@@ -246,3 +242,7 @@
   from = "/ecodev-grants"
   to = "/ecodev-grants/apply"
   force = true
+[[redirects]]
+  from = "/academic-grants-2023/"
+  to = "/academic-grants/"
+  force = true
```

Comment:
Redirect necessary until we publish the new round next year.

Reply:
```
[[redirects]]
  from = "/academic-grants/apply/"
  to = "/academic-grants/"
  force = true
```

More a question about future maintenance, but will we use this redirect once this grant round is over? In addition, would we add:

```
[[redirects]]
  from = "/academic-grants-2023/apply/"
  to = "/academic-grants/"
  force = true
```